plotly/tools.py

```diff
@@ -2347,7 +2347,7 @@ def create_distplot(hist_data, group_labels,
         :param (list[list]) hist_data: Use list of lists to plot multiple data
             sets on the same plot.
         :param (list[str]) group_labels: Names for each data set.
-        :param (float) bin_size: Size of histogram bins. Default = 1.
+        :param (list[float]) bin_size: Size of histogram bins. Default = [1.]
```

Comment:
Re: https://github.com/plotly/plotly.py/pull/455/files#r63061447

Then the type should be `(list[float]|float)`